### PR TITLE
Creating the IPR doc page

### PR DESCRIPTION
**Pull request by remidhum**
In an effort to continue the Robot documentation standardization, the IPR is being moved from the "Real robots" and having its own dedicated page based on the ones from the Pioneer robots.

Preview: https://www.cyberbotics.com/doc/guide/using-the-ipr-robot?version=enhancement-ipr

This branch should be merged after #633.

### CODE REVIEW

**Comment by remidhum** [comment]
- [ ] complete the `IPR models` header based on the Aibo ERS7 and Nao pages

**Comment by fabienrohrer** [comment]
👍

File: guide/using-the-ipr-robot.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+## Using the IPR robot
+
+%figure "IPR"
+
+![ipr.png](images/ipr.png)
+
+%end
+
+The IPR robot is a 6-DOF robotic arm equipped with a gripper composed of 2 rotational fingers.
+It is developed by [Neuronics](http://www.neuronics.be/).
```
Comment (omichel) [inline guide/using-the-ipr-robot.md]: This link is wrong! Not Neuronics was a Swiss company which doesn't exist any more, not a Belgian company... 😆

File: guide/using-the-ipr-robot.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Four models are supported in Webots: HD6M180, HD6Ms180, HD6M90 and HD6Ms90.
+
+More information is available on the Neuronics official [webpage](http://www.neuronics.be/).
```
Comment (omichel) [inline guide/using-the-ipr-robot.md]: No information about IPR arms are available there... Did you find some?

**Comment by omichel** [comment]
We should be very careful about the font and font size used on images. They should be consistent across images and @remidhum defined a rule for that. It seems this rules was not respected here.

**Comment by omichel** [comment]
Also, it is a pity that on the main image, the gripper is not very well understandable, it doesn't look like a gripper. Maybe it's because of the lighting or viewpoint or we should show it opened...

**Pull request merged**